data/json/monster_factions.json

```diff
@@ -338,7 +338,7 @@
     "base_faction": "insect",
     "neutral": [ "centipede", "bee", "wasp" ],
     "hate": [ "insect", "animal" ],
-    "by_mood": "spider"
+    "by_mood": [ "zombie", "fungus" ]
   },
```

Comment:
There are a couple of defaults here that you are likely not aware of. By default, faction members are friendly to each other, but "by_mood" toward other factions. So removing "spider" from "by_mood" means that all the subgroups of spiders will be friendly to each other. Similarly, you don't need "by_mood" for "fungus", since that's the default and nothing else has overridden that.

The faction system has some confusing aspects, I fully realize (including that it currently isn't applied correctly in many cases). With cata_test (or equivalent for windows compiled), you can take a look at the factions by running the `generate_monfactions_attitude_matrix` "test" - that will produce a file with the faction relationships.

Reply:
This is some golden info. Thanks!